src/Appwrite/Platform/Workers/Messaging.php

```diff
@@ -104,7 +113,7 @@ private function processMessage(Database $dbForProject, Document $message): void
         */
         $results = batch(\array_map(function ($providerId) use ($providers, $message, $dbForProject) {
             return function () use ($providerId, $providers, $message, $dbForProject) {
-                $provider = $dbForProject->getDocument('providers', $providerId);
+                $provider = Authorization::skip(fn () => $dbForProject->getDocument('providers', $providerId));
```

Comment:
We shouldn't need to skip here, auth is disabled globally in workers

Reply:
removed this